src/lib/styles/mixins/_effect.scss

```diff
@@ -1,4 +1,9 @@
 @mixin ripple-effect($background, $color: null) {
+  & {
```

Comment:
What does `& {` do any why is it necessary?

Reply:
`& {` is equals to opting out to the waring - i.e. preserving current behavior.

```
To keep the existing behavior, move the declaration above the nested
rule. To opt into the new behavior, wrap the declaration in `& {}`.
```

I can add a TODO but resolving the conflict that way in a mixin that is used in several places was more straightforward.

We can also take more time and check any usages if you prefer?

Reply:
Updated the description.

Reply:
Don't we already opt in to the new behavior by increasing the version of Sass that we're using?

Reply:
And why do we actually want to opt in to the new behavior in these 2 cases?

> We can also take more time and check any usages if you prefer?

If you're not confident that the changes are correct, then we should check.

Reply:
> And why do we actually want to opt in to the new behavior in these 2 cases?

That way you don't have to go through all components that use the mixin and resolve manually the warning by modifying the order of the declarations and the rules.

> If you're not confident that the changes are correct

I'm confident it's correct. `& {}` has the same result as the style on `main`. 

```
.something {
   color: black;
}

===

.something {
   & {
      color: black;
   }   
}
```